fastq_filtrator.py

```diff
@@ -0,0 +1,37 @@
+def read_file(input_fastq):
+    with open(input_fastq) as file:
+        content = file.readlines()
+    reads = []
+    for i in range(0, len(content) - 3, 4):
+        reads.append(list(content[i:i + 4]))
+    return reads
+
+def filter_by_gc(read, gc_bounds):
+    if len(gc_bounds) == 1:
+        gc_bounds = (0, gc_bounds)
+    return gc_bounds[0] <= (sum([1 for c in read if c in 'GC']) * 100) / len(read) <= gc_bounds[1]
+    
+def filter_by_quality(read, quality_threshold):
+    return 100*(1 - 10**(-(sum([ord(r) - ord('!') for r in read]) / len(read))/10)) >= quality_threshold
```

Comment:
Это что-то оооочень странное
```suggestion
    return sum([ord(r) - ord('!') for r in read]) / len(read) >= quality_threshold
```